shaders/cake/cake.vert

```diff
@@ -0,0 +1,21 @@
+#version 450
+
+layout(location = 0) in float left;
+layout(location = 1) in float right;
+layout(location = 2) in int start;
+layout(location = 3) in int end;
+layout(location = 4) in int buffer_index;
+
+layout(location = 0) out float v_left;
+layout(location = 1) out float v_right;
+layout(location = 2) out int v_start;
+layout(location = 3) out int v_end;
+layout(location = 4) out int v_buffer_index;
+
```

Comment:
@Kaydax said that passing all of this to the vertex shader was slow (IDK graphics programming, that's just what he said)